### PR TITLE
When removing a user, I can select a replacement #266

### PR DESCRIPTION
**Pull request by Lengocduong**

### CODE REVIEW

File: docs/_locale/de/LC_MESSAGES/settings.po [hunk 1]
```diff
@@ -40,9 +40,39 @@ msgstr "Die Nutzerliste zeigt alle Mitglieder in Ihrer Organisation. Sie können
...
 msgstr "Administratoren können Nutzer auch löschen indem Sie auf den Papierkorb neben dem Nutzernamen klicken."
 
+msgid "If you want to delete a user account, for example, because the user is leaving your organization, unfinished tasks to this account may still exist. In this case, specify a replacement who will take over open task assignements. The substitute will then have exactly all access rights to complete these tasks, but not automatically inherits all rolls or rights of the deleted user."
+msgstr "Wenn Sie ein Benutzerkonto löschen möchten, da der Nutzer beispielsweise Ihre Organisation verlässt, könnten noch unerledigte Aufgaben vorliegen. In diesem Fall haben Sie die Möglichkeit, einen Vertreter zu bestimmen, der diese offenen Aufgaben übernehmen soll. Der Vertreter erhält dann genau die Zugriffsrechte, die er für die Erledigung der Aufgabe benötigt, erbt jedoch nicht automatisch Rollen und Rechte des gelöschten Nutzers."
+
+msgid "The substitute takes over"
+msgstr "Der Vertreter übernimmt"
+
+msgid "in workflows: owner, assignments, candidates, default values for form fields, transition conditions, JavaScript test values, access"
```
Comment (SilentGert) [inline docs/_locale/de/LC_MESSAGES/settings.po]: Do you usually call it `access` or `access rights`. The same in german `Zugriff` vs. `Zugriffsrechte`.

File: docs/settings.rst [hunk 2]
```diff
@@ -31,9 +31,35 @@ click the *Promote to administrator* button.
...
+------------
+
 Administrators can also delete users,
 using the delete button next to the user name.
 
+If you want to delete a user account, for example,
+because the user is leaving your organization, unfinished tasks to this account may still exist.
+In this case, specify a replacement who will take over open task assignements.
+The substitute will then have exactly all access rights to complete these tasks,
+but not automatically inherits all rolls or rights of the deleted user.
```
Comment (SilentGert) [inline docs/settings.rst]: `rolls` should be `roles`
I'm a bit confused by the last part:
> but not automatically inherits all rolls or rights of the deleted user.

What exactly would be roles and rights?
It is correct that if you use roles within a case for tasks, the value there isn't changed. However, we adjust access rights in cases and tasks where necessary.

Another thing to mention would be the assignment for closed tasks is not changed for audit reasons.

File: docs/settings.rst [hunk 3]
```diff
@@ -31,9 +31,35 @@ click the *Promote to administrator* button.
...
+
+The substitute takes over
+
+* in workflows: owner, assignments, candidates, default values for form fields, transition conditions, JavaScript test values, access
```
Comment (SilentGert) [inline docs/settings.rst]: As mentioned before, I would extend `access` to `access rights`

**Review by SilentGert — CHANGES_REQUESTED** [review]

File: docs/_locale/de/LC_MESSAGES/settings.po (hunk 1 above)
Comment (Lengocduong) [reply docs/_locale/de/LC_MESSAGES/settings.po]: I think, access rights is more precise

File: docs/settings.rst (hunk 2 above)
Comment (Lengocduong) [reply docs/settings.rst]: I 'd like to point out that the substitute will merely have exactly the necessary access right to complete the tasks, but will not automatically inherit all rights of his predecessor.

Comment (SilentGert) [reply docs/settings.rst]: If rights means access rights, it's not correct. The substitute will inherit the access rights for open tasks and cases. If rights means group memberships then that's correct, they are not transferred.

**Review by SilentGert — APPROVED** [review]

**Comment by Lengocduong** [comment]
@SilentGert : Merge please :-) !

**Comment by SilentGert** [comment]
What about the failing travis checks? Do you ignore them?

**Comment by TimKam** [comment]
I'll have a look at the failing checks.

**Review by TimKam — APPROVED** [review]

**Pull request merged**